docs/guides/test-runner/typescript.md

```diff
@@ -0,0 +1,39 @@
+# Test Runner >> Testing TypeScript || 70
+
+If you write your source files in TypeScript, you can test directly from sources without 
+compiling using `tsc`. Add `esbuildPlugin({ ts: true })` to your `web-test-runner.config.js`
+file.
+This uses esbuild to [transform TS sources on-the-fly](https://esbuild.github.io/api/#transform-api).
+Esbuild is not TypeScript, so
+[there are some caveats to using esbuild for testing](https://esbuild.github.io/content-types/#typescript-caveats).
```

Comment:
nit: let's update the link to include `caveats when using TypeScript with esbuild` as formulated in its docs.